src/main/java/org/mule/extension/validation/internal/CommonValidationOperations.java

```diff
@@ -251,6 +257,25 @@ public void isTime(String time, @Optional String locale, @Optional String patter
     validateWith(new TimeValidator(time, nullSafeLocale(locale), pattern, context), context);
   }
 
+  /**
+   * Validates that a ${@code time} did not occur more than ${@code expireAmount} amount of ${@code expireUnits} units
+   * after the current time.
+   *
+   * @param time the time to validate
+   * @param expireAmount the interval size
+   * @param expireUnit the interval unit (as a ${@link ChronoUnit})
+   * @param options the {@link ValidationOptions}
+   * @param config the current {@link ValidationExtension} that serves as config
+   */
+  @Throws(ExpirationErrorType.class)
+  public void expirationTime(LocalDateTime time, Long expireAmount, ChronoUnit expireUnit,
```

Comment:
the term expiration shouldn't be a part of this operation name. Expiration check is one possible use case of this validator. What this is actually doing is checking that an X amount of time has passed since a given time. Rename this as `<validation:is-elapsed time="5" timeUnit="SECONDS" since="#[vars.someDate]" />`